### PR TITLE
Add openra chapter

### PR DESCRIPTION
**Pull request by joostverdoorn**
This PR adds the OpenRA chapter to the book

### CODE REVIEW

**Comment by joostverdoorn** [comment]
@rogierslag :shipit:

**Comment by joostverdoorn** [comment]
@pevers @jabbink @besuikerd FYI

File: chapters/openra/index.md [hunk 1]
```diff
@@ -0,0 +1,477 @@
...
+A summary of the project's stakeholders can be found in [Table 1](#t1).
+
+<a name="t1"></a>
+
+ Type | Entities |
+|---|---|
+| Developers | [@pchote](https://github.com/pchote) , [@Mailaender](https://github.com/Mailaender), [@obrakmann](https://github.com/obrakmann), [@chrisforbes](https://github.com/chrisforbes) |
+| Users | hundreds of players around the world, [ModDB](http://www.moddb.com/games/openra) |
+| Suppliers | .NET/Mono, [Westwood](http://nl.wikipedia.org/wiki/Westwood_Studios), [Electronic Arts](http://www.ea.com/) |
+| Assessors | Preventing copyright infringement done by [@pchote](..) and [@mailaender](..) in https://github.com/openra/openra/issues?q=legal |
```
Comment (rogierslag) [inline chapters/openra/index.md]: These links do not work

Comment (joostverdoorn) [inline chapters/openra/index.md]: They do. However, for you convenience, I've added them anyway :camel: :dromedary_camel:

**Comment by rogierslag** [comment]
Seems perfect :+1: 

@avandeursen :smile:

**Comment by avandeursen** [comment]
Awesome guys, thanks!

**Pull request merged**